util/crs_rules_check/rules-check.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import glob
+import msc_pyparser
+import difflib
+
+class bcolors(object):
+    BLUE      = '\033[94m'
+    CYAN      = '\033[96m'
+    GREEN     = '\033[92m'
+    MAGENTA   = '\033[95m'
+    YELLOW    = '\033[93m'
+    RED       = '\033[91m'
+    BOLD      = '\033[1m'
+    UNDERLINE = '\033[4m'
+    ENDC      = '\033[0m'
+
+    @staticmethod
+    def print_colors():
+        """a helper function to see the colors"""
+        print(bcolors.MAGENTA,   "MAGENTA",   bcolors.ENDC)
+        print(bcolors.BLUE,      "BLUE",      bcolors.ENDC)
+        print(bcolors.CYAN,      "CYAN",      bcolors.ENDC)
+        print(bcolors.GREEN,     "GREEN",     bcolors.ENDC)
+        print(bcolors.YELLOW,    "YELLOW",    bcolors.ENDC)
+        print(bcolors.RED,       "RED",       bcolors.ENDC)
+        print(bcolors.BOLD,      "BOLD",      bcolors.ENDC)
+        print(bcolors.UNDERLINE, "UNDERLINE", bcolors.ENDC)
+
+def print_ok(s = None):
+    if s is None:
+        print(bcolors.GREEN, "[OK]", bcolors.ENDC)
+    else:
+        print(bcolors.GREEN, "%s" % (s), bcolors.ENDC)
+
+def print_err(s = None):
```

Comment:
I think it will be way more useful to print in github actions errors. Take a look at https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#setting-an-error-message.

For example, showing this:
`echo "::error file=util/crs_rules_check/examples/test3.conf,line=1,col=5,endColumn=7::Bad case `endsWith`"`

Will show the error file and your description in a clear nice way in github actions. What do you think?

Of course, you can keep the output we have now, and just print this way if we pass a new flag like `./rules_check.py --output github ....` or similar, and the default will be "console".

Reply:
@airween Is this output planned, or will it be ready for next version?

Reply:
> @airween Is this output planned, or will it be ready for next version?

Yes, it's planned to change - I've already started it this weekend.

Reply:
`--output` has added in f030d2b9. @fzipi please review it again - and thanks for the idea.